scripts/geth-genesis.json

```diff
@@ -1,6 +1,6 @@
 {
   "config": {
-    "chainId": 9000,
+    "chainId": 9002,
```

Comment:
why 9002?

Reply:
we decided to go for 9002 for testing suites. So it uses the `aevmos` base denom  and all the default params and is different than mainnet.